project/Build.scala

```diff
@@ -134,7 +134,7 @@ object Build extends Build {
     testOptions in Test := Seq(Tests.Filter(x => !performanceFilter(x))),
     testOptions in PerformanceTest := Seq(Tests.Filter(x => performanceFilter(x))),
     fork in PerformanceTest := false
-  ) ++ net.virtualvoid.sbt.graph.Plugin.graphSettings ++ publishSettings ++ VersionManagement.newSettings
+  ) ++ VersionManagement.newSettings // ++ publishSettings 
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Whitespace at end of line](https://www.codacy.com/app/flavian/phantom_2/file/2347799639/issues/source?bid=2847375&fileBranchId=3216756#l137)